<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZGNEP_Main</class>
 <widget class="QMainWindow" name="ZGNEP_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1374</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ZGNEP_Main</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,1,5,1,3">
       <item row="0" column="0">
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_image">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
            <property name="text">
             <string>头像</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>117</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_dateTime">
            <property name="font">
             <font>
              <pointsize>25</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image:url(:/main/src/icon/文字底图.jpg);
background-repeat: no-repeat;
background-position: center;
background-attachment: fixed;
background-size: contain;</string>
            </property>
            <property name="text">
             <string>时间日期</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>116</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="label_state">
            <property name="font">
             <font>
              <pointsize>24</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background:#16A085;color:#FFFFFF;border-radius:5px;</string>
            </property>
            <property name="text">
             <string>当前状态:   休息</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="4">
        <widget class="QPushButton" name="zgenp_main_text">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/记事本.png</normaloff>:/main/src/icon/记事本.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="ZGNEP_main_pushBtn_setting">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/设置.png</normaloff>:/main/src/icon/设置.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="zgenp_main_calc">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/计算器.png</normaloff>:/main/src/icon/计算器.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="ZGNEP_main_pushBtn_video">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/播放器.png</normaloff>:/main/src/icon/播放器.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="ZGNEP_main_pushBtn_demonstrate">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/pdf.png</normaloff>:/main/src/icon/pdf.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="ZGNEP_main_pushBtn_transport">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/文件传输助手.png</normaloff>:/main/src/icon/文件传输助手.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="zgnep_main_ppt">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/ppt.png</normaloff>:/main/src/icon/ppt.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QPushButton" name="ZGNEP_main_pushBtn_audio">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/音乐播放器.png</normaloff>:/main/src/icon/音乐播放器.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="zgnep_main_photo">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::PreventContextMenu</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/图库.png</normaloff>:/main/src/icon/图库.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="ZGNEP_main_pushBtn_file">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/文件系统.png</normaloff>:/main/src/icon/文件系统.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="ZGNEP_main_pushBtn_whiteBoard">
         <property name="font">
          <font>
           <pointsize>18</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/白板 .png</normaloff>:/main/src/icon/白板 .png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="zgnep_main_word">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/Word.png</normaloff>:/main/src/icon/Word.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QPushButton" name="zgnep_main_course">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/课程表.png</normaloff>:/main/src/icon/课程表.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="6">
        <widget class="QPushButton" name="zgnep_main_calender">
         <property name="styleSheet">
          <string notr="true">QPushButton {
    /* 设置按钮的样式 */
    border: none; /* 移除按钮的边框 */
    background-color: transparent; /* 设置按钮的背景为透明 */
    padding: 0; /* 移除按钮的内边距 */
}

QPushButton::icon {
    /* 设置图标填充整个按钮 */
    width: 100%; /* 图标宽度占据整个按钮宽度 */
    height: 100%; /* 图标高度占据整个按钮高度 */
    background-size: cover; /* 图标大小自适应按钮大小 */
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="ZGNEP_SRC.qrc">
           <normaloff>:/main/src/icon/日历.png</normaloff>:/main/src/icon/日历.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>60</width>
           <height>60</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1374</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionSetting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ZGNEP_SRC.qrc">
     <normaloff>:/main/src/icon/设置.png</normaloff>:/main/src/icon/设置.png</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="toolTip">
    <string>系统设置</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ZGNEP_SRC.qrc"/>
 </resources>
 <connections/>
</ui>
